alexia/apps/organization/models.py

```diff
@@ -113,6 +114,8 @@ def has_iva(self):
 
     def tended_count(self):
         return BartenderAvailability.objects.filter(
+            Q(event__kegs__gt=0) | Q(event__consumptionform__isnull=False) | Q(event__orders=True) |
```

Comment:
Certainly missing something like a distinct here. Next to that, this does not work on the users-page, as there it checks on a membership

Reply:
Resolved, and tested with @bramvandartel